napari_animation/easing.py

```diff
@@ -0,0 +1,285 @@
+"""
+The easing functions here were adapted from
+https://gist.github.com/zeffii/c1e14dd6620ad855d81ec2e89a859719
+"""
```

Comment:
```suggestion
original c code:
https://raw.githubusercontent.com/warrenm/AHEasing/master/AHEasing/easing.c
Copyright (c) 2011, Auerhaus Development, LLC
http://sam.zoy.org/wtfpl/COPYING for more details.
"""
```
Just adding this, but all the license is maximally permissive.

Link above takes you too
```
            DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
                    Version 2, December 2004

 Copyright (C) 2004 Sam Hocevar <sam@hocevar.net>

 Everyone is permitted to copy and distribute verbatim or modified
 copies of this license document, and changing it is allowed as long
 as the name is changed.

            DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
   TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION

  0. You just DO WHAT THE FUCK YOU WANT TO.
```

Reply:
hahahah fantastic - looks great!